src/GraphQL/Execution/ExecutionHelper.cs

```diff
@@ -10,65 +10,6 @@ namespace GraphQL.Execution
     /// </summary>
     public static class ExecutionHelper
```

Comment:
Only two methods left in `ExecutionHelper`. I wouldn't say they belong to "execution" stage 100%.

Reply:
Right.  `CoerceValue` is also used by the validation context.  `GetArgumentValues` is used mainly by `ReadonlyResolveFieldContext`, but also `SubscriptionExecutionStrategy`.